modules/core/04-channel/keeper/packet.go

```diff
@@ -33,7 +33,7 @@ func (k Keeper) SendPacket(
 		return 0, sdkerrors.Wrap(types.ErrChannelNotFound, sourceChannel)
 	}
 
-	if channel.State == types.CLOSED {
+	if channel.State != types.OPEN {
 		return 0, sdkerrors.Wrapf(
 			types.ErrInvalidChannelState,
 			"channel is CLOSED (got %s)", channel.State.String(),
```

Comment:
Should we maybe change the error text to something like `channel is not OPEN...`?